deprecation/README.md

```diff
@@ -2,4 +2,17 @@
 
 * `Deprecated` PyPI package is used for deprecation
 * annotate deprecated functions with `@depracted` decorator, optionally with message
-* functions will be removed after 6 months from the release where they were marked as deprecated
+* functions will be removed after 6 months from the release in which they were marked as deprecated
+
+## Choice of library
+
+Looked into the options suggested by @lachmanfrantisek which were:
+
+* `Deprecated`
+  * seems as a good choice, offers decorator that has optional parameters such as version or custom message
```

Comment:
Some code example?..;)

(You can show here that you spent some time playing with that.)